assets/js/autocomplete.ts

```diff
@@ -15,17 +15,21 @@ import {
   TermSuggestion,
 } from './utils/suggestions';
 
-type InputFieldElement = HTMLInputElement | HTMLTextAreaElement;
+type AcEnabledInputElement = HTMLInputElement | HTMLTextAreaElement;
 
-let inputField: InputFieldElement | null = null,
-  originalTerm: string | undefined,
-  originalQuery: string | undefined,
-  selectedTerm: TermContext | null = null;
+function hasAcEnabled(element: unknown): element is AcEnabledInputElement {
```

Comment:
Similarly I don't like "autocomplete" being abbreviated, so rename this (and all similar occurences) to `hasAutocompleteEnabled`.